RijsMuseum/app/src/main/java/com/example/rijsmuseum/viewmodel/DetailsViewModel.kt

```diff
@@ -1,10 +1,23 @@
 package com.example.rijsmuseum.viewmodel
 
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.example.network.getters.CollectionsDetailsGetter
+import com.example.network.models.CollectionsDetails
 
 class DetailsViewModel: ViewModel() {
-    fun getCollectionsDetailsRequest(data: CollectionsDetailsGetter.DataReadyCallback) {
-        CollectionsDetailsGetter().getCollectionsDetailsRequest(data)
+    var _dataTitle = MutableLiveData<String>()
+    val _cList = MutableLiveData<CollectionsDetails.ArtObject>()
+
+    fun getCollectionsDetailsRequest(
+        objectNumber: String
+    ) {
+        CollectionsDetailsGetter().getCollectionsDetailsRequest(
+            object : CollectionsDetailsGetter.DataReadyCallback {
+                override fun onDataReady(data: CollectionsDetails.ArtObject) {
+                    _cList.value = data
+                    _dataTitle.value = _cList.value?.title
```

Comment:
why do you have 2 liveDatas if you passed the whole data anyway with the _cList.value = data?